core-blocks/image/index.js

```diff
@@ -165,6 +170,10 @@ export const settings = {
 	save( { attributes } ) {
 		const { url, alt, caption, align, href, width, height, id } = attributes;
 
+		const classes = classnames( align ? `align${ align }` : null, {
+			'is-resized': !! width || !! height,
```

Comment:
Minor: `classnames` doesn't require the value to be explicitly a boolean, so this could have been just `width || height`. Or, if a boolean is desired, coerce after the fact: `Boolean( width || height )`